spec/support/suspenders.rb

```diff
@@ -12,9 +12,10 @@ def create_tmp_directory
   def run_suspenders(arguments = nil)
     Dir.chdir(tmp_path) do
       Bundler.with_clean_env do
-        ENV['TESTING'] = '1'
-
-        %x(#{suspenders_bin} #{APP_NAME} #{arguments})
+        `
```

Comment:
They both work. As far as I could tell, when the suspenders binary is running, it will use the `PATH` set here (and disregards the other, I think because it's a different thread or bash session, not sure). But for tests that don't actually run suspenders, like the one's in the adapter, then `add_fakes_to_path` comes into play.

If there's a better way to do this, I'll take it.